pom.xml

```diff
@@ -17,13 +17,10 @@
   <properties>
     <revision>2.7.2</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.277.2</jenkins.version>
-    <java.level>8</java.level>
+    <jenkins.version>2.387.3</jenkins.version>
```

Comment:
See [the documentation](https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/).